sycl/include/sycl/info/info_desc.hpp

```diff
@@ -58,7 +58,6 @@ enum class device_type : pi_uint64 {
   // TODO: figure out if we need all the below in PI
   custom = PI_DEVICE_TYPE_CUSTOM,
   automatic,
-  host,
```

Comment:
The [SYCL specification](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#appendix.device.descriptors) still has this, even if we don't intend to use it.

Reply:
"host" device_type is brought back https://github.com/intel/llvm/pull/14258/commits/09f3a1cb082665b41cfaabc349c38f4183ca8cf3